src/shortcuts/indent.tsx

```diff
@@ -33,9 +33,13 @@ const indentShortcut: Shortcut = {
   overlay: {
     keyboard: moveCursorForward.keyboard,
   },
+  multicursor: moveCursorForward.multicursor,
```

Comment:
In this case we should just duplicate the config, because we don't want this tightly coupled to `moveCursorForward`.